presto-kudu/src/main/java/com/facebook/presto/kudu/KuduClientConfig.java

```diff
@@ -114,15 +115,27 @@ public KuduClientConfig setDisableStatistics(boolean disableStatistics)
         return this;
     }
 
-    public String getTenant()
+    public String getInferSchemaPrefix()
     {
-        return tenant;
+        return isInferSchemaEnabled() ? inferSchemaPrefix : null;
     }
 
-    @Config("kudu.session.tenant")
-    public KuduClientConfig setTenant(String tenant)
+    @Config("kudu.inferSchemaPrefix")
+    public KuduClientConfig setInferSchemaPrefix(String prefix)
     {
-        this.tenant = tenant;
+        this.inferSchemaPrefix = prefix;
+        return this;
+    }
+
+    public boolean isInferSchemaEnabled()
+    {
+        return inferSchema;
+    }
+
+    @Config("kudu.inferSchema")
```

Comment:
Convention for properties is more like `kudu.infer-schema`

btw, since you handle schemas read/write, "infer" is not the best word. I like "emulate" better, but won't insist.